code/android-core-library/src/phone/java/com/adobe/marketing/mobile/services/ui/AEPMessage.java

```diff
@@ -127,24 +131,10 @@ class AEPMessage implements FullscreenMessage {
     @SuppressLint("ResourceType")
     @Override
     public void show() {
-        if (messagesMonitor != null && messagesMonitor.isDisplayed()) {
-            Log.debug(
-                    ServiceConstants.LOG_TAG,
-                    TAG,
-                    "Message couldn't be displayed, another message is displayed at this time.");
-            fullScreenMessageDelegate.onShowFailure();
-            return;
-        }
-
-        if (!fullScreenMessageDelegate.shouldShowMessage(this)) {
-            Log.debug(
-                    ServiceConstants.LOG_TAG,
-                    TAG,
-                    "Message couldn't be displayed, FullscreenMessageDelegate#shouldShowMessage"
-                            + " states the message should not be displayed.");
-            return;
-        }
+        show(true);
+    }
 
+    public void show(final boolean withDelegateControl) {
```

Comment:
Is this method meant to be public? If so, it is missing in the FullScreenMessage interface.

Reply:
my mistake, this shouldn't be public.